styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+.app {
+    width: 100vw;
+    height: 100vh;
+    background: url(images/background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.nav-bar-list {
+    display: flex;
+    gap: 32px;
+   align-items: center;
+}
+.nav-bar-list-right {
+    display: flex;
+    gap: 12px;
+   align-items: center;
+}
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    justify-content: space-between;
+    margin: auto;
+}
+.nav-link {
+    list-style: none;
+}
+
+.nav-link-text {
+    font-family: 'Inter', sans-serif;
```

Comment:
For something that is global like the `font-family` in this context.
you can put it within the CSS reset. that way you can have some DRY CSS :smile: 
> the color as well, I see it's repetitive

Then for the font weight, it's OK to remove since the default is 500 (looking at the related HTML `link`)
if it's not something like a paragraph or some BOLD Headline that needs some spacing the `line-height` is not really necessary.

- [ ] Please remove `font-weight` and `line-height`